### PR TITLE
fix(windows): install and load wintun.dll from a well-known path instead of setting the current directory

### LINKED ISSUE #3425: bug(windows): can't find where the production exe from CD runs are installing wintun.dll
**Describe the bug**
The production exes are running, but I don't see wintun.dll being installed in %LOCALAPPDATA% or anywhere. So I'm not sure where they're getting it from.

**To Reproduce**
Install the MSI and run Firezone.

**Expected behavior**
Expected wintun.dll to show up in Program Files, ProgramData, or %LOCALAPPDATA5.

**Screenshots / Logs**
N/A

**Platform (please complete the following information):**
 - Firezone Version 86165825a1b1
 - OS: Windows 11

**Additional context**
Related to #3140

### PR DESCRIPTION
**Pull request by ReactorScram**
closes #3425

```[tasklist]
- [x] Switch to connlib-shared for BUNDLE_ID and stuff
- [x] Break out small things into other PRs if possible
- [x] Fix merge conflicts
```

### CODE REVIEW

File: rust/windows-client/src-tauri/src/client/crash_handling.rs [hunk 1]
```diff
@@ -80,8 +86,10 @@ fn start_server_and_connect() -> anyhow::Result<(minidumper::Client, std::proces
         }
 
         server = Some(
+            //
```
Comment (jamilbk) [inline rust/windows-client/src-tauri/src/client/crash_handling.rs]: Typo?

**Review by jamilbk — COMMENTED** [review]
LGTM will wait for @conectado's approval / comments on code organization.

File: rust/windows-client/src-tauri/src/client/crash_handling.rs (hunk 1 above)
Comment (ReactorScram) [reply rust/windows-client/src-tauri/src/client/crash_handling.rs]: Yeah, removed that. I was probably about to write `// TODO: pass the socket name to the server so we don't have to generate it in two places` and then just implemented that and left the slashes.

File: rust/windows-client/firezone-windows-common/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+firezone-windows-common
```
Comment (conectado) [inline rust/windows-client/firezone-windows-common/README.md]: I'd prefer if we use `connlib-shared` rather than adding a new crate.

Comment (conectado) [reply rust/windows-client/firezone-windows-common/README.md]: we can add a windows specific module

Comment (ReactorScram) [reply rust/windows-client/firezone-windows-common/README.md]: Cool, I'll revise this PR

**Review by conectado — APPROVED** [review]

**Pull request merged**